optimum/exporters/executorch/integrations.py

```diff
@@ -88,30 +88,46 @@ def prepare_export_inputs(self):
         # 1. Get export inputs
         model_id = self.model.config.name_or_path
         processor = AutoProcessor.from_pretrained(model_id)
-        sample_conversation_with_audio = [
-            {
-                "role": "user",
-                "content": [
-                    {
-                        "type": "audio",
-                        "url": "https://huggingface.co/datasets/eustlb/audio-samples/resolve/main/dude_where_is_my_car.wav",
-                    },
-                ],
-            }
-        ]
-        processed_inputs = apply_chat_template_with_fallback(
-            processor,
-            sample_conversation_with_audio,
-            add_generation_prompt=True,
-            tokenize=True,
-            return_dict=True,
-            return_tensors="pt",
-        )
+
+        if model_id.startswith("ibm-granite/granite-speech"):
```

Comment:
there could be other fine-tuned granite model besides ibm domain..

What about something like this?
```
  if hasattr(self.model.config, 'encoder_config') and \
     self.model.config.encoder_config.model_type == 'granite_speech_encoder':
 ```